src/components/JobCardForApplicant/index.jsx

```diff
@@ -15,6 +15,8 @@ import { ApplicationCreatedMessage } from './ApplicationCreatedMessage'
 const { publicRuntimeConfig } = getConfig()
 const { optriSpaceContractAddress, blockchainViewAddressURL } =
   publicRuntimeConfig
+const AVG_PRICE_API_URL =
```

Comment:
I think we can extract this logic into `useConversionRate` hook, for example and then re-use it everywhere. WDYT?